packages/babel-preset-expo/babel.config.js

```diff
@@ -1,11 +1,2 @@
-module.exports = function (api) {
-  api.cache(true);
-  return {
-    // Only transpile our source code
-    include(filename) {
-      // Ignore tests and the `unknown` name used in tests
-      return !filename.match(/samples/) && filename.match(/src\/(.*)\.tsx?/);
-    },
-    presets: ['@expo/babel-preset-cli'],
-  };
-};
+// @generated by expo-module-scripts
+module.exports = require('expo-module-scripts/babel.config.cli');
```

Comment:
In this case, the tsconfig is used to transpile the package and the babel.config.js is used for testing. We use the `include` statement to prevent this babel config from running on the test cases, since it's gone, the tests are failing.

Reply:
Why aren't we testing with the same system that builds it? Seems to be a recipe for uncaught issues 😄

Reply:
We are, the babel functions internally shouldn't be using preset CLI code in addition to babel-preset-expo though.